src/main/java/org/spongepowered/api/text/message/Message.java

```diff
@@ -221,21 +222,19 @@
      */
     interface Score extends Message {
 
-        // TODO use Score
-
         /**
-         * Returns the score displayed by this {@link Message}.
+         * Returns the {@link org.spongepowered.api.scoreboard.Score} displayed by this {@link Message}.
          *
-         * @return The score in this message
+         * @return The {@link org.spongepowered.api.scoreboard.Score} in this message
          */
         @Override
-        Object getContent();
+        org.spongepowered.api.scoreboard.Score getContent();
```

Comment:
The plain score does not work here because its overwridden by the class defenition, isnt it?